src/units/abilities.cpp

```diff
@@ -1793,7 +1792,7 @@ namespace
 		bool no_value_weapon_abilities_check =  abilities_list::no_weapon_number_tags().count(tag_name) != 0 || abilities_list::ability_no_value_tags().count(tag_name) != 0;
 		if(filter.has_attribute("cumulative") && no_value_weapon_abilities_check)
 			return false;
-		if(filter.has_attribute("value") && no_value_weapon_abilities_check)
+		if(filter.has_attribute("value") && no_value_weapon_abilities_check && tag_name != "berserk")
```

Comment:
TODO - improve review comment

Needs a comment explaining why Berserk is different.